ansibleplaybookgrapher/cli.py

```diff
@@ -349,6 +357,26 @@ def post_process_args(self, options: Namespace) -> Namespace:
         if self.options.open_protocol_handler == "custom":
             self.validate_open_protocol_custom_formats()
 
+        # create list of roles to exclude
+        if options.exclude_roles:
+            exclude_roles = set()
+            for arg in options.exclude_roles:
+                path = Path(arg)
+                if path.exists():
+                    if path.is_dir():
+                        raise IsADirectoryError(f"{path} is a directory")
```

Comment:
Multiple things:

- We should provide a better error message here.
- More importantly, the roles can be stored in different [places](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html) and I can have a role named after a folder in the current directory. This will fails in this case while running the playbook would work.

Reply:
I see. I fix this bug by allowing directory paths as arguments for the option. If the path exists it will assume that the basename of the given path is the role, which should be excluded. Idk if that makes sense. Let me know your opinion :)